courses/machine_learning/deepdive2/feature_engineering/solutions/1_bqml_basic_feat_eng_bqml.ipynb

```diff
@@ -0,0 +1,1706 @@
+{
```

Comment:
<p>Same story here, the learning objectives should be unique to each lab so setting up the environment and creating the training data don't need to be included.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/777/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
Removed set up from the learning objectives.